checker/src/main/java/org/checkerframework/checker/nullness/NullnessTransfer.java

```diff
@@ -55,6 +61,9 @@
     /** Annotations of the non-null type system. */
     protected final AnnotationMirror NONNULL, NULLABLE;
 
+    /** Java's Map interface. */
```

Comment:
Maybe note that the qualifiers in this type don't matter - it is not used as a fully-annotated ADT, just for asSuper.